app/src/main/java/com/example/tudee/presentation/screen/category/tasks/CategoryTasksScreen.kt

```diff
@@ -66,64 +66,93 @@ fun CategoryTasksScreen(
             }
         }
     }
-
-    Box(modifier = modifier.fillMaxSize()) {
+    TudeeScaffold(
+        showTopAppBar = true,
+        topAppBar = {
+            TopAppBar(
+                onBackButtonClicked = { navController.navigateUp() },
+                showBackButton = true,
+                title = uiState.categoryTasksUiModel?.title,
```

Comment:
Can this be passed as null? and if so, should the user see `null` for the category name instead of an error message?